owlpost/owls.py

```diff
@@ -148,13 +179,23 @@ def match_input(connection, label, category):
         choices[count] = (key, val)
         count += 1
 
+    if exact_match:
+        if choices:
+            for key, val in choices.items():
+                number, lab = val
+                if lab.lower() == label.lower():
+                    match = number
+                    return match
+        match = None
+        return None
+
     index = -1
     if choices:
         for key, val in choices.items():
             number, label = val
             print(str(key) + ': ' + label + ' (' + number +')\n')
 
-        index = int(input("Do any of these match your input? (if none, write -1): "))
+        index = input("Do any of these match your input? (if none, write -1): ")
```

Comment:
Did you test this line and are you running it in Python3? I remember I had to specifically switch it to using int(). Since you would have copied our owls back in, maybe you never had this change in your version.

Reply:
I am sorry. I did not test for the condition when it’s -1. Yes, I am running on Python3.

Reply:
Okay I am going to go ahead and accept the pull request but let's both correct that line ASAP.

Reply:
Thank you. I'll update it on my end.